.golangci.yaml

```diff
@@ -1,15 +1,17 @@
-run:
-  timeout: 10m
-linters-settings:
-  funlen:
-    lines: 75
+issues:
```

Comment:
FWIW this file was coped from [what's in Syft](https://github.com/anchore/syft/blob/main/.golangci.yaml)